### PR TITLE
Helm chart for AGIMUS to run in kubernetes

### PR DESCRIPTION
**Pull request by mathew-fleisch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +39,15 @@ make docker-stop
 UPDATE users SET score=42069, spins=420, jackpots=69, wager=25, high_roller=1 WHERE id=1;
 ```
 
-## Permissions
+### Kubernetes Usage
+
+AGIMUS can also be deployed in kubernetes. The provided helm chart includes a persistent volume claim for mysql to run in a pod, and the agimus container itself. 
+
+```bash
+
```
Comment (zmattingly) [inline README.md:47]: Should there be some code here?

File: README.md [hunk 2]
```diff
@@ -52,8 +64,11 @@ Instructions for how to do this are available through this video at the 58 secon
 
 The bot now requires `Intents.members` and `Intents.presences`.  You must enable this through the "Privileged Gateway Intents" page on the Application page of the Discord developer's portal.
 
+![AGIMUS permissions](https://i.imgur.com/rcZnQCo.png)
 
-## Commands
+![AGIMUS "intents"](https://i.imgur.com/bGl9pf9.png)
```
Comment (zmattingly) [inline README.md:69]: Nice

**Review by zmattingly — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mathew-fleisch) [reply README.md:68]: Whoops. good eye. updated

File: README.md (hunk 2 above)
Comment (mathew-fleisch) [reply README.md:92]: :fingerguns:

**Pull request merged**